lib/pure/times.nim

```diff
@@ -1656,6 +1990,13 @@ proc parse*(value, layout: string, zone: Timezone = local()): DateTime =
     # Otherwise convert to `zone`
     result = dt.toTime.inZone(zone)
 
+proc parseTime*(value, layout: string, zone: Timezone = local()): Time =
```

Comment:
Same as I've written above. Remove this function.

Reply:
Same as above, I definitely disagree if we want this API to be usable

Reply:
Again, agree with andreaferretti. Argumentation is same as above. Here i decided against parse[T] because i feel this as a little too much. But with the concrete in/out i see no opposition for this func's existance

Reply:
This function is less dangerous, but I still think it's unwarranted. ``parseTime(value, layout)`` vs. ``parse(value, layout).toTime()``.

Reply:
It's just a wrapper. Wrappers are used to not overuse your own memory. Yes, it's done with a single line. But remembering (or even auto-completing) `parseTime` is simpler than `parse.toTime`. May be difference is minimal, it still is.
I may remove it from pr, but i definitely disagree.